src/dodal/devices/i18/table.py

```diff
@@ -0,0 +1,28 @@
+from ophyd_async.core import (
+    AsyncStatus,
+    StandardReadable,
+)
+from ophyd_async.epics.motor import Motor
+from pydantic import BaseModel
+
+
+class Four_D_Position(BaseModel):
+    x: float
+    y: float
+    z: float
+    theta: float
+
+
+class Table(StandardReadable):
+    def __init__(self, prefix: str = "", name: str = "") -> None:
```

Comment:
```suggestion
    def __init__(self, prefix: str, name: str = "") -> None:
```

prefix should be a required argument

Reply:
ok